client/grpc_query.go

```diff
@@ -56,8 +56,8 @@ func (ctx Context) Invoke(grpcCtx gocontext.Context, method string, req, reply i
 	// As a result, we direct them to the ABCI flow where they get syncronized.
 	_, isSimulationRequest := req.(*tx.SimulateRequest)
 	isTendermintQuery := strings.Contains(method, "tendermint")
-
-	isGRPCAllowed := !isTendermintQuery && !isSimulationRequest
+	isConcurrentQuery := ctx.Concurrency
+	isGRPCAllowed := !isTendermintQuery && !isSimulationRequest && !isConcurrentQuery
```

Comment:
Grpc should be allowed given that `isConcurrentQuery` is set to true but  `isTendermintQuery` is false and `isSimulationRequest` is false


```suggestion
	isGRPCAllowed := !isTendermintQuery && !isSimulationRequest && isConcurrentQuery
```

Reply:
Good catch, thanks man